google/cloud/spanner_v1/session.py

```diff
@@ -358,11 +358,7 @@ def run_in_transaction(self, func, *args, **kw):
                 raise
             else:
                 if self._database.log_commit_stats:
-                    self._database.logger.info(
-                        "Transaction mutation count: {}".format(
-                            txn.commit_stats.mutation_count
-                        )
-                    )
+                    self._database.logger.info(txn.commit_stats)
```

Comment:
I wnet over this in my other comment, but I'll repeat it here too: The docs for Python Logging say they expect the main `msg` argument to be a string. If you want to add a custom object, see the `extra` argument

Reply:
Ah, that's exactly what I want. I've updated the call to use a formatted message _and_ pass the `CommitStats` via the extra argument. Thank you for your patience while I learn how to use loggers correctly :+1:

Reply:
No problem! Yeah this now LGTM from a logging perspective